src/multi.jl

```diff
@@ -0,0 +1,77 @@
+# Univariate
+
+const FillVectorOfUnivariate{
+    S <: ValueSupport,
+    T <: UnivariateDistribution{S},
+    Tdists <: Fill{T, 1},
+} = VectorOfUnivariate{S, T, Tdists}
+
+function FillDist(dist::UnivariateDistribution, N::Int)
+    return Product(Fill(dist, N))
+end
+FillDist(d::Normal, N::Int) = MvNormal(fill(d.μ, N), d.σ)
+function Distributions.logpdf(
+    dist::FillVectorOfUnivariate,
+    x::AbstractVector{<:Real},
+)
+    return _flat_logpdf(dist.v.value, x)
+end
+function Distributions.logpdf(
+    dist::FillVectorOfUnivariate,
+    x::AbstractMatrix{<:Real},
+)
+    return _flat_logpdf_mat(dist.v.value, x)
+end
+function _flat_logpdf(dist, x)
```

Comment:
Sorry, I'm confused, why is this special implementation and `flatten` needed?

Reply:
Broadcasting `logpdf` with a vector of normal distributions of `TrackedReal` means gives a vector of `TrackedReal`s. Flattening the parameters of the distribution and broadcasting on the parameters directly e.g. `f.(mean::TrackedReal, std::TrackedReal, x::AbstractVector)` gives a `TrackedArray`. This part of the code is really hackish and ugly but it does the trick. I can probably use the `vcat` trick here instead. This part of the code was written before I figure out the `vcat` workaround for `TrackedArray`s.

Reply:
Hmm the `vcat` trick will actually be slower than the current hack. I can use the `vcat` as the fallback though to avoid arrays of `TrackedReal`s there too.

Reply:
I will consider this resolved for lack of a better alternative.